fleet/deploy/render.sh

```
@@ -0,0 +1,75 @@
+# must never be guessed.
+: "${LOOP_INTERVAL:=60}"
+: "${CLUSTER_MANAGER:=agent-sandbox-fleet}"   # matches setup-hub.sh
+: "${HUB_NAMESPACE:=fleet-system}"
+: "${SANDBOX_CAPACITY:=200}"
+export LOOP_INTERVAL CLUSTER_MANAGER HUB_NAMESPACE SANDBOX_CAPACITY
+
+# Everything the file actually references, minus what is now set.
+missing=()
+while read -r var; do
+  [[ -n "${!var:-}" ]] || missing+=("$var")
+done < <(grep -oh '\${[A-Za-z_][A-Za-z0-9_]*}' "$FILE" | tr -d '${}' | sort -u)
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)